CHANGELOG.md

```diff
@@ -3,14 +3,69 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0.0dev - [date]
+## v1.0.0dev
+
+**The content below is the unaltered changelog of the unreleased 2020 version of the pipeline.**
+
+## v0.1.0dev - [date]
 
 Initial release of nf-core/kmermaid, created with the [nf-core](https://nf-co.re/) template.
 
 ### `Added`
 
+- Add option to use Dayhoff encoding for sourmash.
+- Add `bam2fasta` process to kmermaid pipeline and flags involved.
+- Add `extract_coding` and `peptide_bloom_filter` process and flags involved.
+- Add `track_abundance` feature to keep track of hashed kmer frequency.
+- Add social preview image
+- Add `fastp` process for trimming reads
+- Add option to use compressed `.tgz` file containing output from 10X Genomics' `cellranger count` outputs, including `possorted_genome_bam.bam` and `barcodes.tsv` files
+- Add samtools_fastq_unaligned and samtools_fastq_aligned process for converting bam to per cell
+  barcode fastq
+- Add version printing for sencha, bam2fasta, and sourmash in Dockerfile, update versions in environment.yml
+- For processes translate, sourmash compute add cpus=1 as they are only serial ([#107](https://github.com/nf-core/kmermaid/pull/107))
+- Add `sourmash sig merge` for aligned/unaligned signatures from bam files, and add `--skip_sig_merge` option to turn it off
+- Add `--protein_fastas` option for creating sketches of already-translated protein sequences
+- Add `--skip_compare option` to skip `sourmash_compare_sketches` process
+- Add merging of aligned/unaligned parts of single-cell data ([#117](https://github.com/nf-core/kmermaid/pull/117))
+- Add renamed package dependency orpheum (used to be known as sencha)
```

Comment:
Some sentence end with point and some other not, might be better to be consistent